transaction-view/src/result.rs

```diff
@@ -0,0 +1,3 @@
+#[derive(Debug, PartialEq, Eq)]
+pub struct TransactionParsingError;
```

Comment:
clippy made me have **some** error type instead of just using `()`.
Thought about using `Option`, but `Result` seems more appropriate in general here